orange/src/main/java/com/hartwig/hmftools/orange/algo/OrangeAlgo.java

```diff
@@ -106,6 +107,8 @@ public class OrangeAlgo
     @NotNull
     private final List<DriverGene> driverGenes;
     @NotNull
+    private final Map<String, String> signaturesEtiologies;
```

Comment:
Maybe a more correct name would be "etiologyPerSignature". That makes it more clear your keys are signatures and the values are the etiologies?